### PR TITLE
MM-52818 - create config setting to enable/disable playbooks product

### PR DESCRIPTION
**Pull request by sbishel**
#### Summary
Creates config setting to allow playbooks to be disabled in product mode.

#### Ticket Link

  Fixes https://mattermost.atlassian.net/browse/MM-52818

#### Release Note

```release-note
Adds config setting to enable/disable playbooks in product mode
```

### CODE REVIEW

**Review by crspeller — APPROVED** [review]

**Comment by amyblais** [comment]
@mgdelacroix @wiggin77 A kind note that v8.0 feature complete is scheduled for June 14th.

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

File: server/config/client_test.go [hunk 1]
```diff
@@ -326,6 +326,20 @@ func TestGetClientConfig(t *testing.T) {
 				"ExperimentalSharedChannels": "true",
 			},
 		},
+		{
+			"Disable playbooks",
```
Comment (mgdelacroix) [inline server/config/client_test.go:330]: nit: should this be "Enable playbooks"?

**Review by mgdelacroix — APPROVED** [review]
LGTM! :+1:

File: server/config/client_test.go (hunk 1 above)
Comment (sbishel) [reply server/config/client_test.go]: Update to "Default Playbooks Enabled"

**Review by esethna — APPROVED** [review]

**Comment by sbishel** [comment]
@lindalumitchell - I'm not sure there is a way to test this until it is merged. I don't think you can update the config.json file in spinwick. There is no UI, so it needs updated and restarted.

**Review by lindalumitchell — APPROVED** [review]
Sounds good, thanks Scott!

**Pull request merged**